src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -162,10 +174,11 @@ private static DesktopElement ElementFromUIAElement(IUIAutomationElement uia, Re
         /// <summary>
         /// Get DesktopElements from UIAElements.
         /// </summary>
-        /// <param name="uia"></param>
         /// <returns>An IEnumerable of <see cref="DesktopElement"/></returns>
-        private static IEnumerable<DesktopElement> ElementsFromUIAElements(IList<IUIAutomationElement> elementList, Registrar registrar)
+        private IEnumerable<DesktopElement> ElementsFromUIAElements(IList<IUIAutomationElement> elementList, DesktopDataContext dataContext)
         {
+            EnsureContextConsistency(dataContext);
```

Comment:
Looks like this is only called from `ElementsFromProcessId`, which already has a check, so this is redundant

Reply:
Fixed